packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -4,6 +4,18 @@ title: Okta API Products Release Notes
 
 ## March
 
+### Weekly Release 2021.03.2
+
+| Change                                                    | Expected in Preview Orgs |
+| --------------------------------------------------------- | ------------------------ |
+| [Bugs fixed in 2021.03.2](#bugs-fixed-in-2021-03-2)       | March 17, 2021           |
+
+#### Bugs fixed in 2021.03.2
+
+* After updating a group name using the [Groups API](/docs/reference/api/groups/#update-group), the change wasn't reflected in the target application with [**Group Push**](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Directory_Using_Group_Push) enabled. (OKTA-375190)
+
+* When creating a user with a recovery question using an OAuth access token rather than an API token, an invalid session error was returned. (OKTA-361888)
```

Comment:
```suggestion
* When creating a User with a recovery question using an OAuth access token rather than an API token, an invalid session error was returned. (OKTA-361888)
```